src/scss/components/_scenarios.scss

```diff
@@ -0,0 +1,106 @@
+.scenarios {
+  $main: &;
+
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 230px;
+  padding-top: 30px;
+  position: relative;
+  width: 100%;
+
+  &__title {
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-title;
+    height: 54px;
+    line-height: 30px;
+    margin-bottom: 102px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    width: 227px;
+  }
+
+  &__principal-container {
+    display: flex;
```

Comment:
The same as before, flex is not taking effect here so please remove the properties `display: flex;`, `flex-direction: row-reverse;`, `justify-content: center;` and `margin: auto;`

Reply:
Done